addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -9200,7 +9201,7 @@ registry.ContainerWidth = SnippetOptionWidget.extend({
      */
     selectClass: async function (previewMode, widgetValue, params) {
         await this._super(...arguments);
-        if (previewMode === 'reset') {
+        if (previewMode === 'reset' || !previewMode) {
             this.$target.removeClass('o_container_preview');
         } else if (previewMode) {
             this.$target.addClass('o_container_preview');
```

Comment:
Easier way? I let you decide.
```js
if (previewMode) {
    this.$target.addClass('o_container_preview');
} else {
    this.$target.removeClass('o_container_preview');
}
```
And if you keep your version -> double quotes "reset"

Reply:
previewMode peut prendre 3 valeurs: `"reset"` `true` `false`. 
avec `if(previewMode)` je catch `"reset"` et `true`, or je ne veux que `true` ;)

Reply:
`if (previewMode === true)` so ?

Reply:
`this.$target[0].classList.toggle('o_container_preview', previewMode === true)`? :wink: